src/fragments/lib/troubleshooting/js/strict-mode.mdx

```diff
@@ -0,0 +1,43 @@
+This section lists known TypeScript errors and their workarounds when [strict mode](/lib/troubleshooting/strict-mode/q/platform/js/) is enabled on your application.
+
+## Declaration file for GraphQL
+
+If you see the error
+
+```
+Could not find a declaration file for module 'graphql/error/GraphQLError'` 
+Could not find a declaration file for module 'graphql/language/ast'` 
+```
+
+Add a declaration file `src/types.d.ts` with the following content to resolve the error:
+
+```ts
+declare module 'graphql/language/ast' { export type DocumentNode = any }
+declare module 'graphql/error/GraphQLError' { export type GraphQLError = any }
+```
+
+## Declaration file for aws-exports
+
+If you see the error
+
+```
+Could not find a declaration file for module './aws-exports'. 'aws-exports.js' implicitly has an 'any' type.
+```
+
+Create a `aws-exports.d.ts` file on the same level as `aws-exports` with the following content:
+
+```ts
+declare const awsmobile: Record<string, any>
+export default awsmobile; 
+```
+
+## Storage Type Error
+
+If you see the error
+
+```
+TS2344: Type 'T[U]' does not satisfy the constraint '(...args: any) => any'
+TS2344: Type 'T[api]' does not satisfy the constraint '(...args: any) => any'.
+```
+
+You can resolve this by setting [`strictNullChecks`](https://www.typescriptlang.org/tsconfig#strictNullChecks) to false within your `tsconfig.json`.
```

Comment:
```suggestion
```
This is being addressed from https://github.com/aws-amplify/amplify-js/pull/8956, removing. Thanks @jamesaucode ✨